examples/CarthageBuildTest/Cartfile.resolved

```diff
@@ -0,0 +1 @@
+github "facebook/AsyncDisplayKit" "eb6aa28d42f3dd547f048d0a4d5cd4787f0647b0"
```

Comment:
No, we don't need this file for the travis tests. It gets regenerated with `carthage update`